malasakit-django/pcari/views.py

```diff
@@ -234,9 +232,10 @@ def personal_information(request):
 @language_selectable
 def quantitative_questions(request):
     questions = QuantitativeQuestion.objects.all()
-    question_text = [(question.id, question.prompt, question.left_text,
-                      question.right_text) for question in questions]
-    context = {'questions': question_text}
+    question_attrs = [(question.id, question.prompt, question.input_type,
+                       question.minval, question.maxval, question.left_text,
+                       question.right_text) for question in questions]
+    context = {'questions': question_attrs}
```

Comment:
Please see my comment on #60 for refactoring this to be more readable. See [Django's documentation](https://docs.djangoproject.com/en/dev/topics/templates/#variables) for how to do this.